shell/components/form/NameNsDescription.vue

```diff
@@ -381,12 +382,18 @@ export default {
     selectNamespace(e) {
       if (!e || e.value === '') { // The blank value in the dropdown is labeled "Create a New Namespace"
         this.createNamespace = true;
-        this.$parent.$emit('createNamespace', true);
+        this.$store.dispatch(
```

Comment:
This solution is great, although should we maybe use an index in case of multiple `CruResource`? I remember we have some cases with more of them together at the same time.

Reply:
I don't think we really want to modify behavior (when avoidable) given that these changes are really only to support getting vue3 in.